helmfile.d/charts/user-rbac/templates/rolebindings/alertmanager.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: alertmanager-configurer
+  namespace: alertmanager
```

Comment:
As we for some reason have an option to change the namespace then this needs to honour it as well.